packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -184,4 +182,15 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 
 		return this.importedNotes[resolvedPath];
 	}
+
+	public isLinkToLocalFile(path: string) {
+		try {
+			new URL(path);
```

Comment:
By the way I've just checked `c:\\test.txt` or `c:/test.txt` and `new URL` doesn't throw an error which means they won't be considered link to files. Not to mention that `file:///path/to/file.txt` is both a valid URL and valid link to a local file. Actually I'm wondering what we're doing here testing for URLs when we are not looking for URLs?

Are there no other reasonable ways to know if something look like a valid local path? I feel like this is a solved problem and there must be plenty of libraries or code snippets out there that can do this. Maybe even in Joplin source code we already have this.